database/seeders/GamesTableSeeder.php

```diff
@@ -47,5 +48,13 @@ public function run(): void
                 'Flags' => AchievementFlag::OfficialCore,
             ]));
         });
+
+        Game::all()->each(function (Game $game) {
+            $game->leaderboards()->saveMany(Leaderboard::factory()->count(random_int(0, 10))->create([
+                'GameID' => $game->ID,
+                'Title' => "Test leaderboard",
+                'Description' => "I am a seeded leaderboard",
```

Comment:
This is a bit too prescriptive -- tons of leaderboards could potentially be generated with the same titles and descriptions.

I believe the default gives us plenty of fuzz:
```php
'Title' => ucwords(fake()->words(2, true)),
'Description' => fake()->sentence(),
```